frontends/main/src/app/styled.tsx

```diff
@@ -9,7 +9,7 @@ import { styled } from "ol-components"
  */
 
 export const PageWrapper = styled.div({
-  height: "calc(100vh - 80px)",
+  height: "100vh",
```

Comment:
Fixes the gap at the bottom of the page:

<img width="400" alt="image" src="https://github.com/user-attachments/assets/c48ff876-eced-43bd-a962-03764725ed4e">